packages/boot/test/fast-usdc/snapshots/fast-usdc.test.ts.md

```diff
@@ -17,10 +17,10 @@ Generated by [AVA](https://avajs.dev).
         {
           chainPolicies: {
             Arbitrum: {
+              attenuatedCttpBridgeAddress: '0xe298b93ffB5eA1FB628e0C0D55A43aeaC268e347',
```

Comment:
This address is different than the previous `nobleContractAddress`, should we update it in multichain-testing/test/fast-usdc/config.ts as well?

Reply:
Good catch I am basing this on top of master, but I'll make sure to make this change there too